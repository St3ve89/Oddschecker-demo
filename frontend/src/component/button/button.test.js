import { Button } from '.';
import renderer from 'react-test-renderer';

test('renders Button component', () => {
  const button = renderer.create(<Button />).toJSON();
  expect(button).toMatchSnapshot();
});
