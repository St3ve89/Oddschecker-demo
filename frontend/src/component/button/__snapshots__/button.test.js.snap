// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Button component 1`] = `
<button
  className="buttonstyle__CustomButton-sc-175iax6-0 cFjAcq"
>
  <span />
</button>
`;
